js/witness_calculator.js

```diff
@@ -395,7 +395,7 @@ class WitnessCalculatorCircom2 {
         this.n32 = this.instance.exports.getFieldNumLen32();
 
         this.instance.exports.getRawPrime();
-        const arr = new Array(this.n32);
+        const arr = new Uint32Array(this.n32);
```

Comment:
Thanks for submitting this. For context, here's the circom2 code: https://github.com/iden3/circom/blob/v2.1.6/code_producers/src/wasm_elements/common/witness_calculator.js#L116